src/main/java/seedu/address/logic/commands/AddAppointmentCommand.java

```diff
@@ -24,6 +25,7 @@ public class AddAppointmentCommand extends Command {
             + PREFIX_PATIENT_ID + "PATIENT_ID "
             + PREFIX_DATETIME + "DATETIME "
             + PREFIX_ATTEND + "ATTEND "
+            + PREFIX_FEEDBACK_SCORE + "FEEDBACK_SCORE "
             + PREFIX_APPOINTMENT_DESCRIPTION + "APPOINTMENT DESCRIPTION\n"
             + "Example: " + COMMAND_WORD + " "
```

Comment:
Please update the example command to reflect feedback score as well.

Reply:
Done